From 45beb7073a1cef89e87a2d562ce8c233fc140dae Mon Sep 17 00:00:00 2001
From: "Daniel M. Capella" <polyzen@users.noreply.github.com>
Date: Sun, 22 Jul 2018 22:21:01 +0000
Subject: [PATCH] manpage: fixup mistaken show playlist/track-list shortcuts

This was mistaken in 496b13227b7f4b47a660bbf4e314f9a55b7e8867 and
not noticed in review.
---
 DOCS/man/mpv.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/DOCS/man/mpv.rst b/DOCS/man/mpv.rst
index c99a30e11f..4ddcc6811e 100644
--- a/DOCS/man/mpv.rst
+++ b/DOCS/man/mpv.rst
@@ -210,11 +210,11 @@ Alt + and Alt -
 Alt+BACKSPACE
     Reset the pan/zoom settings.
 
-F9
+F8
     Show the playlist and the current position in it (useful only if a UI window
     is used, broken on the terminal).
 
-F10
+F9
     Show the list of audio and subtitle streams (useful only if a UI window  is
     used, broken on the terminal).
 
